3711/2/commands/checkout.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require_relative '../authentification'
+require_relative '../handlers/photo'
+
+module CheckOutCommand
+  include Authentification
+  include PhotoHandler
+
+  def checkout!(*)
+    if registered?
```

Comment:
this `if` is super complex because of nested `if else` condition
consider refactor this